common/cmake/devices.cmake

```diff
@@ -1048,6 +1206,8 @@ function(ADD_FPGA_TARGET)
   set_target_properties(${NAME} PROPERTIES
       TOP ${TOP}
       BOARD ${BOARD}
+      OUT_LOCAL ${OUT_LOCAL}
```

Comment:
I don't believe there should be any reason to have either OUT_LOCAL or OUT_LOCAL_REL?  Should probably get using `get_file_location` over using `${OUT_LOCAL}`?